internal/pkg/deploy/cloudformation/app.go

```diff
@@ -569,6 +694,17 @@ func (cf CloudFormation) deleteStackSetInstances(name string) error {
 	return cf.appStackSet.WaitForOperation(name, opID)
 }
 
+func (cf CloudFormation) deleteStackSetInstance(name, account, region string) error {
+	opId, err := cf.appStackSet.DeleteInstance(name, account, region)
+	if err != nil {
+		if IsEmptyErr(err) {
+			return nil
+		}
+		return err
+	}
+	return cf.appStackSet.WaitForOperation(name, opId)
```

Comment:
Just making sure - if there is no instance in the said account + region, i take that we will still get an operation ID. What happens when we wait for that empty operation? Is that an instant return?

Reply:
Yeah, it instantly returns "succeeded". CFN views it as a successful delete even though nothing was deleted.